supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

```diff
@@ -31,6 +31,20 @@
 
 *See also*:
 
+[discrete]
+[[permission]]
+==== image:images/yes.png[yes] permission (noun)
+*Description*: A "permission" is a property of an object. The term is usually used in the context of filesystems and their objects, such as files and directories.
```

Comment:
@bergerhoffer What's the decision for definitions? I recall that you suggested quotation marks and italics, but not sure if it was decided using this format.

Reply:
The quotation marks are consistent with other entries currently in the SSG. Marking this comment as resolved (for now)